src/tribler/gui/i18n/ru_RU.ts

```diff
@@ -632,7 +632,7 @@ If unsure, press &apos;No&apos;. You will be able to remove those directories fr
     </message>
     <message>
         <location filename="../tribler_window.py" line="649"/>
-        <source>Version %s of Tribler is available.Do you want to visit the website to download the newest version?</source>
+        <source>Version %s of Tribler is available. Do you want to visit the website to download the newest version?</source>
```

Comment:
Maybe you can add the correct translation for this message as well here.

Reply:
It is already here. I can confirm that it is correct.